tests/foreman/cli/test_repository.py

```diff
@@ -1895,8 +1893,7 @@ def test_positive_srpm_list_end_to_end(self, repo, target_sat):
         assert len(srpm_list) == 1
         assert target_sat.cli.Srpm.info({'id': srpm_list[0]['id']})[0]['filename'] == SRPM_TO_UPLOAD
         assert (
-            int(target_sat.cli.Repository.info({'id': repo['id']})['content-counts']['source-rpms'])
-            == 1
+            int(target_sat.cli.Repository.info({'id': repo['id']})['content-counts']['srpms']) == 1
```

Comment:
just for optimizing this line, it'd save extra line
```suggestion
     assert target_sat.cli.Repository.info({'id': repo['id']})['content-counts']['srpms'] == '1'
```

Reply:
I recommend integral value assertions exclusively in this test to maintain consistency.

Reply:
Similar change could be done on ~20 other places in that module. I will keep the consideration to whoever takes the repositories comp eval (probably @vijaysawant) and keep it as is for consistency with the rest of the module.